### ISSUE TITLE
[VPAT] Contrast of Feedback Text and Button Control

### ISSUE THREAD

**Issue by pbailie** [Submitty/Submitty#5136]
In the Grade Reports view, a feedback text message is positioned too close to a button control.  Those with vision impairment may have difficulty discerning the text message due to insufficient 
contrast differences.  Please review screenshot where marked by the red arrow.
![Screen Shot 2020-03-06 at 5 46 44 PM](https://user-images.githubusercontent.com/12703624/76132646-f4596100-5fe1-11ea-83c4-be0e7aa2a640.png)
